### DISCUSSION TITLE
Disable personal project

### DISCUSSION THREAD

**Discussion by markwimpory** [Kong/insomnia#7263] [Ideas] [answered]
In some orgs there will be rules about the use of cloud and storing detail of api's there.  Ideally we would like the ability to disable personal projects preferably at sign-up time, just to avoid the possibility of using cloud storage.

**Comment by subnetmarco** [comment]
@markwimpory yes absolutely - we are actually going to work on this immediately after 9.0 and it will ship in 9.1 in May. There was a technical limitation that prevented us from having the personal project converted to local, but we are close to removing it.

There is a small chance it can make it in 9.0 by EoM, will see.

**Comment by subnetmarco** [comment, marked as answer]
In Insomnia 9.x it is now possible to convert a personal project to local only.
